src/czechpmdevs/buildertools/commands/BlockInfoCommand.php

```diff
@@ -48,11 +48,12 @@ public function execute(CommandSender $sender, string $commandLabel, array $args
         if(BuilderTools::getConfiguration()["items"]["blockinfo-stick"]["enabled"]) {
             $item = VanillaItems::STICK();
             $item->setCustomName(BuilderTools::getConfiguration()["items"]["blockinfo-stick"]["name"]);
-            $item->setNamedTagEntry(new ListTag(Item::TAG_ENCH, [], NBT::TAG_Compound));
-            $item->setNamedTagEntry(new ByteTag("buildertools", 1));
+            $nbt = $item->getNamedTag();
+            $nbt->setByte("buildertools", 1);
+            $nbt->setTag(Item::TAG_ENCH, new ListTag([], NBT::TAG_Compound));
+            $item->setNamedTag($nbt);
+            /*$item->addEnchantment(new EnchantmentInstance(EnchantmentIdMap::getInstance()->fromId(50), 1));*/
```

Comment:
I think we do not need this commented out, github can show us how the code looked like in the past.